examples/gtfs_reader.rs

```diff
@@ -0,0 +1,7 @@
+extern crate csv;
+extern crate navitia_model;
+
+fn main() {
+    let objects = navitia_model::gtfs::read("./examples/data/");
+    println!("Count of networks loaded : {}", objects.networks.len());
```

Comment:
you can dump the model using serde_json

Reply:
I don't understand what you want me to do here. Print out a JSON output for the first network ?

Reply:
That's for you. I don't really care. You can easilly dump the model as json, interesting for testing.

Reply:
ok, good idea. I'll try to use it in the tests. If you're ok with it, I'll let the example like that for the moment.

Reply:
In test, that's easier to directly check the structure.

As you want, that's just for you ;-)